metadata-ingestion/tests/integration/mysql/mysql_mces_no_db_golden.json

```diff
@@ -315,7 +315,7 @@
 },
 {
     "entityType": "dataset",
-    "entityUrn": "urn:li:dataset:(urn:li:dataPlatform:mysql,datacharmer.salaries,PROD)",
+    "entityUrn": "urn:li:dataset:(urn:li:dataPlatform:mysql,dataCharmer.salaries,PROD)",
```

Comment:
Looking into ways to avoid change in dataset urn case for sql_common.

Reply:
Current behavior looks good as schema level (the level immediately before table) has been case-sensitive for other sources as well